File: 0122-best-time-to-buy-and-sell-stock-ii/solution.cpp
class Solution {
public:
    int maxProfit(vector<int>& v) {
        int res = 0 , n = v.size(); 
        for( int i = 1 ; i < n ; i++){
            if(v[i] - v[i-1] > 0)res += v[i] - v[i-1] ;
        }
        return res ;
    }
};

File: 0122-best-time-to-buy-and-sell-stock-ii/solution.java
class Solution {
    public int maxProfit(int[] prices) {

        int n = prices.length ;
        int res = 0 ; 
        int check = prices[n-1];

        for(int i = 1 ; i < n ; i++){
            if((prices[i] - prices[i-1]) > 0 ) res += (prices[i] - prices[i-1]);
        }
        return res ;
    }
}
